"Build Java" in "Lint and build regularly" job failed

The error looks like:
```
Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/mojo/maven-metadata.xml (21 kB at 529 kB/s)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.291 s
[INFO] Finished at: 2024-07-12T13:34:34Z
[INFO] ------------------------------------------------------------------------
Error:  No plugin found for prefix 'spotless' in the current project and in the plugin groups [org.apache.maven.plugins, org.codehaus.mojo] available from the repositories [local (/home/runner/.m2/repository), central (https://repo.maven.apache.org/maven2)] -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/NoPluginFoundForPrefixException
Error: Process completed with exit code 1.
```
See details https://github.com/apache/incubator-pegasus/actions/runs/9908766114/job/27375255558.

--- a/.github/workflows/regular-build.yml
+++ b/.github/workflows/regular-build.yml
@@ -140,9 +140,11 @@ jobs:
       - uses: actions/setup-java@v1
         with:
           java-version: ${{ matrix.java }}
+      - name: Download thrift
+        working-directory: ./java-client/scripts
+        run: ./download_thrift.sh
       - name: Build
         working-directory: ./java-client
         run: |
-          cd scripts && bash recompile_thrift.sh && cd -
           mvn spotless:apply
           mvn clean package -DskipTests

